### DISCUSSION TITLE
Why the 3rd phase issue is never answered ?

### DISCUSSION THREAD

**Discussion by ytan1984** [PaddlePaddle/PaddleSpatial#261] [BaiduKDDCup2022] [answered]
Dear,

I found an interesting fact that none of question about the 3rd phase is answered,
- #254 
- #253 
- #247 
- partial of #230 

well, please help the 3rd phase clear for everyone,
Thanks

@zhoujingbo @ramber1836

**Comment by yangtohsi** [comment]
+10086

**Comment by zhoujingbo** [comment, marked as answer]
From July 16th 00:01 AM to July 18th 11: 59 PM (AOE time), we will go to Phase 3 of the Baidu KDD Cup. During this phase, the test data will be updated. Each team can submit its prediction model three times per day. During this phase, the organizer team will not answer any questions about submission errors (except for the Pending Timeout error) in the discussion forum. The leaderboard in Phase 3 will determine the candidate awardee teams.

**Reply by ytan1984** [reply]
Thanks for the update.

So the 11:59PM is the latest submit time, 
The running submission could be continue as long as it's submitted before that time. 

And we could see each submitted score, so totally we have 9 times

am I right?
Thanks

**Reply by ramber1836** [reply]
Yes, you are right.

**Reply by ytan1984** [reply]
> Yes, you are right.

@ramber1836 

If the total time is 9 times, then it's very strange. Because I submit one time in 20:00 Beijing time in 0716, then I decide remain another 2 time in 0717 before 20:00 Beijing time, but I found in 17th July, the daily quota is reset to 3 times.

So the totally submit time is actually 12 ?
